res/css/views/rooms/_EventTile.scss

```diff
@@ -611,11 +606,6 @@ $left-gutter: 64px;
         mask-size: 80%;
     }
 
-    &.mx_EventTile_e2eIcon_warning,
-    &.mx_EventTile_e2eIcon_normal {
-        opacity: 1;
```

Comment:
Because there is not an E2E icon other than these two, it is redundant to reset the default value with this line.